test/parallel/test-readline-interface-tty-readstream.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/5574
+
+/**
+ * Test that characters from stdin are not randomly lost when using a
+ * `readline.Interface` and a `tty.ReadStream` on `process.stdin`
+ */
+
+var Interface   = require('readline').Interface;
+var Readable    = require('stream').Readable;
+var ReadStream  = require('tty').ReadStream;
+var spawn       = require('child_process').spawn;
+var strictEqual = require('assert').strictEqual;
+
+var common = require('../common');
```

Comment:
This should be `const` also, and should be the first import, immediately following the `'use strict'` pragma